lib/portalUtils.js

```diff
@@ -50,7 +50,7 @@ _.extend(exports, {
     },
     getPom: function(c, p) {
         p = p || '';
-        var pomPath = path.join(c.config._local.path, p, '/pom.xml');
+        var pomPath = path.join(c.config.bbrc.path, p, '/pom.xml');
```

Comment:
It's better then to keep this utils in your experiments either. Without messing up with stable branch.